scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -75,6 +69,17 @@ def test_support_alternative_backends(xp, data, f_name_n_args):
     args_np = [np.asarray(data.draw(npst.arrays(dtype_np, shape, elements=elements)))
                for shape in shapes]
 
+    # Make exceptions for counterexamples
+    if is_jax(xp):
+        if f_name in {'gammainc', 'gammaincc'}:  # google/jax#20507
+            pytest.skip("google/jax#20507")
+        if f_name == 'rel_entr':  # google/jax#21265
+            x, y = args_np[0], args_np[1]
+            assume(not np.any((x == 0) & (y == 1)))
```

Comment:
Oops, I think each arg is 0d, so we probably don't need the `any`/`all` stuff.

While we're here, we might also want to improve this below:

> PyTorch seems to struggle with precision near the poles of `gammaln`...

I think we can make the tolerances tighter and add `assume`s for Pytorch as needed.